Source/Protocols/Request.swift

```diff
@@ -26,6 +26,10 @@ public extension Request where ResponseObject: Collection, ResponseObject.Iterat
 
 public extension Request where ResponseObject == EmptyResponse {
   func parse(_ j: JSON) -> Result<ResponseObject, SwishError> {
+    #if swift(>=3.3)
```

Comment:
How do you feel about outdenting these?

Reply:
do you mean like

```swift
func parse() {
  #if swift(>=3.3)
    return .success(())
  #else
    return .success()
  #endif
}
```
?

Reply:
I'm fine with it, just want to make sure we're on the same page wrt style.

Reply:
Oh I suppose I meant the C style, outdenting to the left edge:

```swift
  func parse() {
#if swift(>=3.3)
    return .success(())
#else
    return .success()
#endif
  }
```

Reply:
That looks... real bad to me tbh. How do you feel about the way I just indented the file?

Reply:
also this is the most bandaid-y of bandaid-y code so we probably don't need to go too deep into this here bikeshed.

Reply:
I'm not too fussed, so feel free to use whichever style you prefer. I think I tend to go for that style to minimise diff noise when the conditionals are eventually removed.

Reply:
yep, I hear you. This is a dead-end refactor though, these changes are never even making it to master. I'm going to go ahead and merge this as is currently written.